spec/services/subscriptions/create_service_spec.rb

```diff
@@ -137,6 +137,15 @@
       end
     end
```

Comment:
Layout/CommentIndentation: Incorrect indentation detected (column 1 instead of 0).